libs/core/src/lib/busy-indicator/busy-indicator-extended/busy-indicator-extended.directive.ts

```diff
@@ -0,0 +1,27 @@
+import { AfterContentInit, Directive, ElementRef } from '@angular/core';
+
+@Directive({
+    // tslint:disable-next-line:directive-selector
+    selector: '[fd-busy-indicator-extended]'
+})
+export class BusyIndicatorExtendedDirective implements AfterContentInit {
+    /** @hidden */
+    constructor(private _eleRef: ElementRef) {}
+    ngAfterContentInit(): void {
+        this._appendCssToParent();
+    }
+
+    /** @hidden */
+    _appendCssToParent(): void {
+        const hasLabel = this.elementRef().nativeElement.firstElementChild.getAttribute('label');
```

Comment:
Can we use contentChild to refer to the component instance?
That will guarantee that we are trying to access the valid field